src/Repository/ImageRepository.php

```diff
@@ -94,20 +95,36 @@ private function findOrCreateFromSource(string $source, ImageOrigin $origin): ?I
 
         try {
             $this->imageManager->store($source, $filePath);
+            $this->logger->debug('stored the image');
```

Comment:
you still need this debug line? If not, you might want to remove it again. Sure, it's behind debug(), but it can get messy fast.

Reply:
true